src/3rdparty/walkontable/src/overlay/left.js

```diff
@@ -146,8 +146,7 @@ class LeftOverlay extends Overlay {
    */
   adjustRootElementSize() {
     const { wtTable, rootDocument, rootWindow } = this.wot;
-    const masterHolder = wtTable.holder;
-    const scrollbarHeight = masterHolder.clientHeight === masterHolder.offsetHeight ? 0 : getScrollbarWidth(rootDocument);
+    const scrollbarHeight = getScrollbarWidth(rootDocument);
```

Comment:
Is name of variable proper?

Reply:
In this context it's ok. I think we can leave it at it is.